test/bugs-reported/bug4.future

```diff
@@ -0,0 +1,28 @@
+bug: The List Module does not support accessing elements of a list based on index. I feel that this should be supported.
```

Comment:
Thanks for splitting this into two much simpler files. Can the bug4 files be removed?